code/game/objects/items/robot/items/tools.dm

```diff
@@ -319,9 +319,10 @@
 
 /obj/item/borg/cyborg_omnitool/medical/reference_item_for_parameters()
 	var/datum/component/butchering/butchering = src.GetComponent(/datum/component/butchering)
-	butchering.butchering_enabled = tool_behaviour == (TOOL_SCALPEL && TOOL_SAW)
+	if(tool_behaviour == (TOOL_SCALPEL && TOOL_SAW))
+		butchering.butchering_enabled = TRUE
```

Comment:
This not now these operators work, if you want to check for two values you need to check it twice. 
```suggestion
	if(tool_behaviour == TOOL_SCALPEL || tool_behaviour == TOOL_SAW))
		butchering.butchering_enabled = TRUE
```
(If you are wondering, what you are actually currently checking is `tool_behavior == TOOL_SAW`, because when you `&&`, the result is actually the last-most truthy value.)

Reply:
Which means you can go back to the original code btw: 
```dm
butchering.butchering_enabled = (tool_behaviour == TOOL_SCALPEL || tool_behaviour == TOOL_SAW)
```